graphql_api/types/me/me.py

```diff
@@ -104,11 +104,7 @@ def resolve_tracking_data(current_user, _, **kwargs):
 @me_bindable.field("termsAgreement")
 @sync_to_async
 def resolve_terms_agreement(current_owner: Owner, _, **kwargs) -> Optional[bool]:
-    try:
-        owner_profile = current_owner.profile
-        return owner_profile.terms_agreement
-    except OwnerProfile.DoesNotExist:
-        return None
+    return current_owner.user.terms_agreement
```

Comment:
just to confirm, an owner will always have a user right?

Reply:
I believe so as this query requires user to be authenticated already.